cryptofeed/backends/influxdb.py

```diff
@@ -53,17 +53,30 @@ def __init__(self, addr: str, db: str, key=None, create_db=True, numeric_type=st
           Create database if not exists
         numeric_type: str/float
           Convert types before writing (amount and price)
+        org: str (For InfluxDB 2.0 compatibility)
+          Orgnaization name for authentication
+        bucket: str (For InfluxDB 2.0 compatibility)
+          Bucket name for authentication
+        token: str (For InfluxDB 2.0 compatibility)
+          Token string for authentication
+        precision: str (For InfluxDB 2.0 compatibility)
+          Precision level among (s, ms, us, ns) 
         """
         super().__init__(addr, **kwargs)
-        self.addr = f"{addr}/write?db={db}"
+        if (org and bucket and token):
+            self.addr = f"{addr}/api/v2/write?org={org}&bucket={bucket}&precision={precision}"
+            self.headers = {"Authorization": f"Token {token}"}
+        else:
+            if create_db:
```

Comment:
in 2.0 do you no longer need to create the db?

Reply:
No, db in the 1.x is replaced with a bucket in 2.0.

Reply:
do you need to create buckets before writing to them?

Reply:
Following this [Influx 2.0 doc](https://v2.docs.influxdata.com/v2.0/organizations/buckets/create-bucket/), we can only create a bucket through a web GUI(it does not mean that http method is available) and CLI command.  

We should have to create bucket before writing, but it cannot be executed by http request.